spec/index.bs

```diff
@@ -1956,11 +1955,14 @@ enum GPUTextureFormat {
 };
 </script>
 
-The depth aspect of the `depth24plus` family of formats
-({{GPUTextureFormat/depth24plus}} and {{GPUTextureFormat/depth24plus-stencil8}})
+The depth aspect of the {{GPUTextureFormat/depth24plus-stencil8}}) format
 may be implemented as either a 24-bit unsigned normalized value ("depth24unorm")
 or a 32-bit IEEE 754 floating point value ("depth32float").
 
+The {{GPUTextureFormat/stencil8}}) format may be implemented as
+either a real "stencil8", or "depth24stencil8", where the depth aspect is
```

Comment:
Or depth32-stencil8?

Reply:
I believe this is required for some Vulkan devices (either `VK_FORMAT_D24_UNORM_S8_UINT` or `VK_FORMAT_D32_SFLOAT_S8_UINT`, there's no requirement for any other stencil formats at all)

Reply:
ouch, that sucks. There is also `depth16stencil8` in Vulkan that can back this, fwiw, but it's also not required